### PR TITLE
[ADP-3397] Bump to node 9.1

### PR DESCRIPTION
**Pull request by paolino**
- [x] Bump node runtime version to 9.1.0
- [x] Remove all local-cluster node configuration discrimination based on current era
- [x] Update memory benchmark node configurations

ADP-3397

### CODE REVIEW

File: justfile [hunk 1]
```diff
@@ -138,19 +138,19 @@ integration-tests match:
     '.#integration-exe' \
     -c integration-exe -j 2 --match="{{match}}"
 
-#start a shell with 9.0.0 in scope
+#start a shell with 9.1.0 in scope
 node-8121:
```
Comment (Anviking) [inline justfile]: The names `node-8121` and `node-900` are now out of date

File: lib/local-cluster/lib/Cardano/Wallet/Launch/Cluster/Node/GenNodeConfig.hs [hunk 2]
```diff
@@ -193,19 +189,15 @@ genNodeConfig nodeSegment name genesisFiles clusterEra logCfg = do
 
 controlExperimental :: ClusterEra -> ChangeValue
 controlExperimental = \case
-    BabbageHardFork -> setExperimental False
-    ConwayHardFork -> setExperimental True
+    _ -> setExperimental True
```
Comment (Anviking) [inline lib/local-cluster/lib/Cardano/Wallet/Launch/Cluster/Node/GenNodeConfig.hs]: Shouldn't this be `False` now?

File: justfile [hunk 3]
```diff
@@ -138,19 +138,13 @@ integration-tests match:
...
-node-8121:
-  nix shell \
-  --accept-flake-config \
-  'github:IntersectMBO/cardano-node?ref=9.0.0#cardano-node' \
-  'github:IntersectMBO/cardano-node?ref=9.0.0#cardano-cli'
 
-# start a shell with 9.0.0 in scope
-node-900:
+# start a shell with 9.1.0 in scope
+node-901:
```
Comment (Anviking) [inline justfile]:
```suggestion
node-910:
```

😛

File: lib/local-cluster/lib/Cardano/Wallet/Launch/Cluster/Node/GenNodeConfig.hs [hunk 4]
```diff
@@ -193,19 +193,15 @@ genNodeConfig nodeSegment name genesisFiles clusterEra logCfg = do
...
 setExperimental :: Bool -> ChangeValue
 setExperimental enabled value = value
     & atKey "ExperimentalProtocolsEnabled" ?~ Bool enabled
     & atKey "ExperimentalHardForksEnabled" ?~ Bool enabled
 
 setHardForksForLatestEras :: ClusterEra -> ChangeValue
-setHardForksForLatestEras clusterEra =
-    case clusterEra of
-        BabbageHardFork -> setHardFork (T.pack $ show BabbageHardFork)
-        ConwayHardFork ->
```
Comment (Anviking) [inline lib/local-cluster/lib/Cardano/Wallet/Launch/Cluster/Node/GenNodeConfig.hs]: Will this change not make the integration tests run on Conway regardless of setting?

Comment (paolino) [reply lib/local-cluster/lib/Cardano/Wallet/Launch/Cluster/Node/GenNodeConfig.hs]: yes, you are totally right . Fixing it ...

**Review by Anviking — APPROVED** [review]
Thanks! 🙌

**Review by paweljakubas — APPROVED** [review]
:tada:

**Pull request merged**